### PR TITLE
Release 0.4.0-rc.1

### PR DESCRIPTION
**Pull request by rartych**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:
This is "Release PR" for Commonalities Release 0.4.0-rc.1

#### Changelog input

```
Commonalities Release 0.4.0-rc.1

```

#### Additional documentation 

https://wiki.camaraproject.org/display/CAM/Meta-release+Process#MetareleaseProcess-Milestoneactivities

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM :)

**Review by maxl2287 — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,11 +2,74 @@
...
+* Updated API versioning guidelines chapter 5 of API Design Guidelines by @tanjadegroot in https://github.com/camaraproject/Commonalities/pull/215
+  
+
+### Fixed
+* API Design Guidelines updated with character set guidance by @trehman-gsma in https://github.com/camaraproject/Commonalities/pull/143
+* Mandated '+' in all phoneNumber formats by @fernandopradocabrillo in https://github.com/camaraproject/Commonalities/pull/148
+* Linting rules problem with Traffic Influence API #161 by @VijayKesharwani in https://github.com/camaraproject/Commonalities/pull/169
+* CAMARA_common.yaml - bugs and typos fix by @fernandopradocabrillo in https://github.com/camaraproject/Commonalities/pull/174
+* API Design Guidelines updated on HTTPs usage by @AxelNennker in https://github.com/camaraproject/Commonalities/pull/205
+* Artifacts aligned with API Design Guidelines for Notification Subscription by @PedroDiez in https://github.com/camaraproject/Commonalities/pull/219
```
Comment (PedroDiez) [inline CHANGELOG.md:62]: Suggestion to include as Fixed PRs:
https://github.com/camaraproject/Commonalities/pull/229
https://github.com/camaraproject/Commonalities/pull/234
https://github.com/camaraproject/Commonalities/pull/241

Comment (PedroDiez) [reply CHANGELOG.md:60]: ok, that is fine. Understood the point of "technical change".

**Review by PedroDiez — APPROVED** [review]
LGTM

**Comment by tanjadegroot** [comment]
For the release process, please add @camaraproject/TSC_participants to the reviewers of this PR.

**Review by shilpa-padgaonkar — APPROVED** [review]
/LGTM

**Comment by hdamker** [comment]
@rartych @camaraproject/commonalities_maintainers:

Within TSC today ([draft minutes](https://wiki.camaraproject.org/display/CAM/2024-07-04+TSC+Minutes)) we have decided to keep the release candidate PRs for Commonalities and ICM open until Wednesday, July 10th, for a final review including the TSC participants. So please wait with the merge until then.

**Review by diegogonmar — APPROVED** [review]
LGTM

**Review by eric-murray — APPROVED** [review]

**Comment by hdamker** [comment]
@camaraproject/tsc-participants thanks for the incoming approvals for the PR. Just adding the team again as GitHub is always happy with the first review/approval from one member of the team and drops it from the reviewer list. 

The review is open until Wednesday, July 10th.

**Comment by patrice-conil** [comment]
LGTM

**Review by patrice-conil — APPROVED** [review]
LGTM

**Review by hdamker — APPROVED** [review]
LGTM

**Review by Kevsy — APPROVED** [review]

**Comment by Kevsy** [comment]
LGTM

**Pull request merged**